chore: Updated CHANGELOG post 1.7 release

roypat opened the pull request:
Move everything that we just released in 1.7 into the 1.7 section, and keep everything we didnt release in the "Unreleased" section.

## Changes

...

## Reason

...

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license. For more information on following Developer
Certificate of Origin and signing off your commits, please check
[`CONTRIBUTING.md`][3].

## PR Checklist

- [ ] If a specific issue led to this PR, this PR closes the issue.
- [ ] The description of changes is clear and encompassing.
- [ ] Any required documentation changes (code and docs) are included in this
  PR.
- [ ] API changes follow the [Runbook for Firecracker API changes][2].
- [ ] User-facing changes are mentioned in `CHANGELOG.md`.
- [ ] All added/changed functionality is tested.
- [ ] New `TODO`s link to an issue.
- [ ] Commits meet
  [contribution quality standards](https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md#contribution-quality-standards).

______________________________________________________________________

- [ ] This functionality cannot be added in [`rust-vmm`][1].

[1]: https://github.com/rust-vmm
[2]: https://github.com/firecracker-microvm/firecracker/blob/main/docs/api-change-runbook.md
[3]: https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md

pb8o commented on CHANGELOG.md:
```diff
@@ -10,6 +10,23 @@ and this project adheres to
 
 ### Added
 
+### Changed
```
commit title has typo CHANGEOG -> CHANGELOG

zulinx86 reviewed (commented):
This is also not released as part of 1.7.
```
- [#4492](https://github.com/firecracker-microvm/firecracker/pull/4492): Changed
  `--config` parameter of `cpu-template-helper` optional. Users no longer need
  to prepare kernel, rootfs and Firecracker configuration files to use
  `cpu-template-helper`.

```

roypat commented:
> This is also not released as part of 1.7.
> 
> ```
> - [#4492](https://github.com/firecracker-microvm/firecracker/pull/4492): Changed
>   `--config` parameter of `cpu-template-helper` optional. Users no longer need
>   to prepare kernel, rootfs and Firecracker configuration files to use
>   `cpu-template-helper`.
> ```

Ahh, true, thanks! fixed :)

zulinx86 commented on CHANGELOG.md:
```diff
@@ -10,6 +10,27 @@ and this project adheres to
+  the `start_time_us` and `start_time_cpu_us` metrics. These parameters were
+  never meant to be used by end customers, and we recommend doing any such time
+  adjustments outside Firecracker.
+- [#4492](https://github.com/firecracker-microvm/firecracker/pull/4492): Changed
+  `--config` parameter of `cpu-template-helper` optional. Users no longer need
+  to prepare kernel, rootfs and Firecracker configuration files to use
+  `cpu-template-helper`.
```
That is not deprecated, but optional :)

roypat replied:
Aaaaaaaaa, so sorry!! Shouldnt be touching PRs before I had coffee >.>

zulinx86 reviewed (approved):
Ship it!
```
      I\
      I \
      I  \
      I*--\
      I    \
      I     \
      I______\
 _____I__O______
  \     ( )     b  ^  ^
^^^^^^^^^^^^^^^^^    ^
```
